static/index.html

```diff
@@ -25,6 +25,10 @@
       <div class="logo">
         <img src="/icons/logo_ocsf.svg">
         <span>Learn OCaml</span>
+        <button id ="feedback" onclick="window.location.href='https://github.com/ocaml-sf/learn-ocaml/issues/new/choose';" target = "_blank">
+          <!-- <span class = "label"> Feedback </span> -->
+          <img src="/icons/icon_feedback.svg" style="width: 22px">
+        </button>
```

Comment:
```suggestion
        <a class="btn" href="https://github.com/learn-ocaml-community/teaching-feedback#readme" target = "_blank">
          <img src="/icons/icon_feedback.svg">
```

(to comply with usual *accessibility rules*: `<a>` is better than `<button>` here, https://a11y-101.com/design/button-vs-link )

**and TODO:** You forgot to commit the .svg image.